server.js

```diff
@@ -39,6 +39,14 @@ app.post('/', function(req, res) {
   //  Construct Message
   var emailMessage = req.body.message;
 
+  // Make sure the email message is not empty
+  if (!emailMessage) {
+    res.send({
```

Comment:
Whoops! Sorry Oamar I missed this on my first pass. Along with the error message, we also want to specify that we are sending back an HTTP error. So we will need to send a `400` status code. If you look above at the error handler for not specifying a valid key, you'll see that in use.

```suggestion
    res.status(400).send({
```